1436790/1438462: Part 33: 20 files with MSDN link fixes.

DennisLee-DennisLee opened the pull request:
Contributes to PR #[2037](https://github.com/dotnet/docs/issues/2037) and applies to the following work items:
- [User Story 1436790: Content Update: update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1436790)
- [Task 1438462: Update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1438462)

DennisLee-DennisLee committed:
Another 20 files with MSDN link fixes.

mairaw commented on docs/framework/wcf/samples/basic-ajax-service.md:
```diff
@@ -6,7 +6,7 @@ ms.assetid: d66d0c91-0109-45a0-a901-f3e4667c2465
 This sample demonstrates how to use Windows Communication Foundation (WCF) to create a basic ASP.NET Asynchronous JavaScript and XML (AJAX) service (a service that you can access using JavaScript code from a Web browser client). The service uses the <xref:System.ServiceModel.Web.WebGetAttribute> attribute to ensure that the service responds to HTTP GET requests and is configured to use the JavaSc ...
   
- AJAX support in WCF is optimized for use with ASP.NET AJAX through the `ScriptManager` control. For an example of using WCF with ASP.NET AJAX, see the [AJAX Samples](https://msdn.microsoft.com/library/f3fa45b3-44d5-4926-8cc4-a13c30a3bf3e).  
+ AJAX support in WCF is optimized for use with ASP.NET AJAX through the `ScriptManager` control. For an example of using WCF with ASP.NET AJAX, see the [AJAX Samples](https://docs.microsoft.com/previous-versions/dotnet/netframework-3.5/bb410780(v=vs.90)).  
```
```suggestion
 AJAX support in WCF is optimized for use with ASP.NET AJAX through the `ScriptManager` control. For an example of using WCF with ASP.NET AJAX, see the [AJAX Samples](ajax.md).  
```

mairaw commented on docs/framework/wcf/samples/custom-binding-imperative.md:
```diff
@@ -75,4 +75,4 @@ Press <ENTER> to terminate client.
   
 ## See also
-- [Custom Binding](https://msdn.microsoft.com/library/657e8143-beb0-472d-9cfe-ed1a19c2ab08)
+- [Custom Binding Samples](https://docs.microsoft.com/previous-versions/dotnet/netframework-3.5/ms751479(v=vs.90))
```
```suggestion
- [Custom Binding Samples](custom-binding.md)
```

mairaw reviewed (changes requested):
Thanks @DennisLee-DennisLee for fixing the links and also the broken sample formatting. Just a couple of nits before merging (there's also a build warning)

mairaw committed:
Apply suggestions from code review

Accepting Maira's 2 autocommittable suggestions. Still need to correct another Ajax samples link manually.

Co-Authored-By: DennisLee-DennisLee <v-dele@microsoft.com>

DennisLee-DennisLee committed:
Merge branch 'master' of https://github.com/dotnet/docs into v-dele-1438462-033

DennisLee-DennisLee committed:
Updated the AJAX samples link. Also removed an empty section.

DennisLee-DennisLee committed:
Fixed a broken link.

mairaw reviewed (approved):
Thanks!
